samples/compile-timeDI/build.sbt

```diff
@@ -0,0 +1,18 @@
+import java.io.File
+import PlayKeys._
+
+name := "j"
```

Comment:
I don't know why we choose `j` as a name but I think it's safe to give a more meaningful name `compile-timeDI` and `runtime-DI` is fine.